lib/msal-common/src/logger/Logger.ts

```diff
@@ -62,7 +62,7 @@ export class Logger {
         const defaultLoggerCallback = () => {};
         this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
         this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = loggerOptions.logLevel || LogLevel.Info;
+        this.level = (typeof(loggerOptions.logLevel) === "number") ? loggerOptions.logLevel : LogLevel.Info;
```

Comment:
Tests?

Reply:
On it, just have to get a little creative since the level attribute is private.

Reply:
Added tests

Reply:
Missed some edge cases, adding them now

Reply:
Did a refactor after adding edge-case tests, validating LogLevel was a bit more complicated than it seemed.